frontend/src/views/ShootItemTerminal.vue

```diff
@@ -15,220 +15,45 @@ limitations under the License.
  -->
 
 <template>
-  <div
-    class="g-droppable-listener"
-    style="height:100%; width:100%; background-color: #333"
-    v-shortkey.once="{bottom: ['ctrl', 'shift', 'h'], right: ['ctrl', 'shift', 'v']}"
-    @dropped="droppedAt"
-    @shortkey="addFromShortkey"
-  >
-    <g-splitpane
-      v-if="splitpaneTree"
-      :splitpaneTree="splitpaneTree"
-      ref="splitpane"
-    >
-      <template v-slot="{item}">
-        <g-terminal
-          :uuid="item.uuid"
-          :data="item.data"
-          @terminated="onTermination(item)"
-          @split="orientation => onSplit(item, orientation)"
-        ></g-terminal>
-      </template>
-    </g-splitpane>
-    <target-selection-dialog ref="targetDialog" :shootItem="shootItem"></target-selection-dialog>
-  </div>
+  <!-- TODO on shootsubscription done do not fetch seedInfo -->
```

Comment:
what about this TODO? Will it be resolved in this PR?

Reply:
no this PR did not change anything in this regards but as @holgerkoser is working on a refactoring in this area I would suggest to leave this TODO for now and follow up once the refactoring is done.
It's just an unnecessary call

Reply:
ok